pyproject.toml

```diff
@@ -148,8 +149,9 @@ pretty = true
 [[tool.mypy.overrides]]
 module = [
     "docker.*",
-    "grpc.*",
     "grpc_health.*",
+    "grpc.*",
+    "networkx",
```

Comment:
We use `networkx` (https://networkx.org/documentation/stable/) to compute an order in which the objects can be stored s.t. their dependencies are all already stored.
It's probably overkill for this simple task, but I image we might also want to do other graph-related things later; I've found it to be a good fit for these in previous projects.

Reply:
Here's the Snyk advisor page on this dependency: https://snyk.io/advisor/python/networkx